src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -702,7 +703,62 @@ public static IEnumerable<object[]> VerifyUploadServersStreamsAndExpectedData
                 }
             }        
         }
-        
+        #endregion
+
+        #region Version tests
+        [ActiveIssue(4754, PlatformID.Windows)]
+        [Theory]
+        [MemberData("Http2Servers")]
+        public async Task SendAsync_RequestVersion10_ResponseVersion10(Uri server)
```

Comment:
I'll move it to stay consistent within this file.  Most of the rest of the repo has it on its own line, hence my tendency.